web/routing.go

```diff
@@ -115,7 +115,7 @@ func newRenderer(assetsPath string) (*renderer, error) {
 func SetupAppsHandler(appsHandler echo.HandlerFunc) echo.HandlerFunc {
 	secure := middlewares.Secure(&middlewares.SecureConfig{
 		HSTSMaxAge:    hstsMaxAge,
-		CSPDefaultSrc: []middlewares.CSPSource{middlewares.CSPSrcSelf, middlewares.CSPSrcParent},
+		CSPDefaultSrc: []middlewares.CSPSource{middlewares.CSPSrcSelf, middlewares.CSPSrcParent, middlewares.CSPSrcWhitelist},
```

Comment:
Well, I'm annoyed.

When we started the whitelist, we have said that it was OK to load images (and maybe other assets, sounds? videos?) from other domains, but we don't want scripts and even less JSON APIs or websockets. But our proxy for these stuff is not here, and you have several apps that need it.

@jinroh, @aenario, @poupotte: what do you think about that?